caravel/static/js/continuation.js

```diff
@@ -50,8 +57,17 @@ var fetchMore = coalesce(function(complete) {
             return;
 
         // Append the results to the existing results <div>
-        var frag = document.createElement("div");
+        if (viewQuery == "v=List") {
+            var frag = document.createElement("table");
+            var att = document.createAttribute("class");
+            att.value = "table listings-table"
+            frag.setAttributeNode(att)
+        }
+        else {
+            var frag = document.createElement("div");
```

Comment:
Even though JavaScript lets you do this (defining `var` inside an if-statement), it feels a little wonky. Maybe move the `var` outside the block?